packages/esbuild/test/plugins/BUILD.bazel

```diff
@@ -0,0 +1,57 @@
+load("//:index.bzl", "generated_file_test", "js_library", "nodejs_binary", "npm_package_bin")
```

Comment:
mention the path to this test folder in the docs somewhere since it serves as an example